.github/workflows/ci.yml

```
@@ -78,8 +76,25 @@ jobs:
         with:
           # We need the parent commit to do a diff
           fetch-depth: 2
+      - name: "Fetching PR description"
+        id: fetch-pr
+        if: github.event_name == 'pull_request'
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+          PR_JSON: pull_request.json
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh api "/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}" > "${PR_JSON}"
+          # It requires delimiter to pass multiline strings through
+          # GITHUB_OUTPUT. Since these are already escaped JSON strings, pass
+          # the JSON strings and later use fromJSON to decode them.
+          echo pr-title=$(jq '.title' "${PR_JSON}") >> "${GITHUB_OUTPUT}"
+          echo pr-body=$(jq '.body' "${PR_JSON}") >> "${GITHUB_OUTPUT}"
```

Probably the  expansion here should be quoted?

```suggestion
          echo pr-title="$(jq '.title' "${PR_JSON}")" >> "${GITHUB_OUTPUT}"
          echo pr-body="$(jq '.body' "${PR_JSON}")" >> "${GITHUB_OUTPUT}"
```

---

Done.